docs/FAQs.md

```diff
@@ -3,10 +3,63 @@ sidebar_position: 7
 ---
 # FAQs
 
-### Contact and Getting Support
+### How can I submit my data?
```

Comment:
Replace "with your email" by either a link to registration docs, or all the methods of registration